### DISCUSSION TITLE
Get Current User from Oqtane Module

### DISCUSSION THREAD

**Discussion by rchibnik** [oqtane/oqtane.framework#1722] [Q&A] [answered]
How can I get the name or object representing the currently logged in user from my Oqtane module.

I can't find an example of this anywhere.

Thanks for your suggestions.

**Comment by leigh-pointer** [comment, marked as answer]
Your module should inherit from ModuleBase
you can then access the PageState.User instance

**Reply by rchibnik** [reply]
Thanks. Thats what I was looking for.
